src/App.js

```diff
@@ -175,6 +178,8 @@ class App extends Component {
   };
 
   toggleLeftDrawer = (isItOpen) => () => {
+    const pathname = this.props.history.location.pathname;
+    reactGAmodalview(pathname);
```

Comment:
what about if isItOpen is false ? are you still tracking a page view ?

Reply:
fixed